plenum/server/consensus/consensus_data_provider.py

```diff
@@ -26,6 +35,9 @@ def set_validators(self, validators: List[str]):
         self._validators = validators
         self._quorums = Quorums(len(validators))
 
+    def set_primary_name(self, primary_name: str):
+        self.primary_name = primary_name
```

Comment:
Given that primary_name is a simple field do we need this function?

Reply:
removed